app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/evaluateCloudFormationChangeSetExecutionMarkerIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { IExecutionMarkerIconProps } from 'core/pipeline/config/stages/common/ExecutionMarkerIcon';
+
+export class EvaluateCloudFormationChangeSetExecutionMarkerIcon extends React.Component<IExecutionMarkerIconProps> {
```

Comment:
Since this is just needed for rendering, you can make it a functional component.

```js
export const EvaluateCloudFormationChangeSetExecutionMarkerIcon = (props: IExecutionMarkerIconProps) => {
    if (...) { return html }
    return newHtml
}
```

Reply:
Done 👍

Reply:
I had to change this back to be a class, it is required by https://github.com/sergiopena/deck/blob/execute_changesets/app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/deployCloudFormationStackStage.ts#L35 when builds the module.